storage/Devices/Bcache.h

```diff
@@ -101,13 +101,22 @@ namespace storage
 	const BcacheCset* get_bcache_cset() const;
 
 	/**
-	 * Attach a caching set to the Bcache.
+	 * Associates a caching set to the Bcache.
 	 *
 	 * This method does not make sense for Flash-only Bcache devices.
 	 *
 	 * @throw LogicException, Exception
 	 */
-	void attach_bcache_cset(BcacheCset* bcache_cset);
+	void add_bcache_cset(BcacheCset* bcache_cset);
```

Comment:
I assume the ABI break here is considered to be OK.

Reply:
Yes, because `attach_bcache_cset` was added during SP1 development phase. So we are still in time to break it.